packages/repository-json-schema/test/unit/build-schema.unit.ts

```diff
@@ -88,24 +88,72 @@ describe('build-schema', () => {
       );
     });
 
-    it('converts types in strings', () => {
-      expect(metaToJsonProperty({type: 'number'})).to.eql({
-        type: 'number',
+    it('converts Boolean', () => {
+      expect(metaToJsonProperty({type: Boolean})).to.eql({
+        type: 'boolean',
+      });
+    });
+
+    it('converts String', () => {
+      expect(metaToJsonProperty({type: String})).to.eql({
+        type: 'string',
       });
     });
 
-    it('converts primitives', () => {
+    it('converts Number', () => {
       expect(metaToJsonProperty({type: Number})).to.eql({
         type: 'number',
       });
     });
 
-    it('converts arrays', () => {
+    it('converts Date', () => {
+      expect(metaToJsonProperty({type: Date})).to.eql({
+        type: 'date',
```

Comment:
I think JavaScript `Date` should be described as `date-time`, because `Date` holds both date *and* time.

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#data-types


Common Name | type | format | Comments
-- | -- | -- | --
date | `string` | `date` | As defined by full-date - [RFC3339](https://xml2rfc.ietf.org/public/rfc/html/rfc3339.html#anchor14)
dateTime | `string` | `date-time` | As defined by date-time - [RFC3339](https://xml2rfc.ietf.org/public/rfc/html/rfc3339.html#anchor14)

LoopBack 3.x describes `Date` properties as `{type: 'string', format: 'date-time'}` too, see loopback-swagger's [lib/specgen/schema-builder.js](https://github.com/strongloop/loopback-swagger/blob/c40420ecb2da2044202dda6d332f46377e318f0d/lib/specgen/schema-builder.js#L103-L106):

```js
    case 'date':
      schema.type = 'string';
      schema.format = 'date-time';
      break;
```